src/main/kotlin/com/jetbrains/micropython/devices/PyboardDeviceProvider.kt

```diff
@@ -0,0 +1,55 @@
+package com.jetbrains.micropython.devices
+
+import com.intellij.execution.configurations.CommandLineState
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.process.OSProcessHandler
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.jetbrains.micropython.run.MicroPythonRunConfiguration
+import com.jetbrains.micropython.settings.MicroPythonFacet
+import com.jetbrains.micropython.settings.MicroPythonTypeHints
+import com.jetbrains.micropython.settings.MicroPythonUsbId
+import com.jetbrains.micropython.settings.microPythonFacet
+import com.jetbrains.python.packaging.PyRequirement
+
+/**
+ * @author stefanhoelzl
+ */
+class PyboardDeviceProvider : MicroPythonDeviceProvider {
+    override val persistentName: String
+        get() = "Pyboard"
+
+    override val documentationURL: String
+        get() = "https://github.com/vlasovskikh/intellij-micropython/wiki/Pyboard"
```

Comment:
Could you please create a page in [my wiki](https://github.com/vlasovskikh/intellij-micropython/wiki) for Pyboard devices listing you as a Pyboard device owner and possibly a maintainer of the Pyboard device type for the plugin?

Reply:
done